server/controllers/events/events_controller.go

```diff
@@ -366,7 +366,8 @@ func (e *VCSEventsController) handleBitbucketCloudPullRequestEvent(w http.Respon
 		e.respond(w, logging.Error, http.StatusBadRequest, "Error parsing pull data: %s %s=%s", err, bitbucketCloudRequestIDHeader, reqID)
```

Comment:
Please document the notes you provided in your above comment into the runatlantis.io directory so it's accessible on the website

Reply:
@nitrocode done